build_metal.sh

```diff
@@ -1,69 +1,34 @@
 #!/bin/bash
-: '
-TLDR: Follow the steps outlined below to build metal. For more in-depth information, keep reading
-
-Steps:
```

Comment:
There was really nothing from here you wanted to keep as comments?

Reply:
Correct, I want a single source of truth on documentation.
Anything worth documenting should be in README.md.